src/main/java/lotto/domain/winpolicy/FiveAndBonusMatch.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain.winpolicy;
+
+import lotto.domain.lotto.Lotto;
+import lotto.domain.lotto.LottoNumber;
+
+public class FiveAndBonusMatch extends Policy {
+  public FiveAndBonusMatch() {
+    super(30_000_000);
+  }
+
+  @Override
+  public Boolean isMatch(Lotto latestWinLotto, Lotto lotto, LottoNumber bonusNumber) {
+    if (latestWinLotto.matchCountOf(lotto).equals(5) && lotto.containLottoNumber(bonusNumber)) {
+      return true;
+    }
+
+    return false;
```

Comment:
```suggestion
    return latestWinLotto.matchCountOf(lotto).equals(5) && lotto.containLottoNumber(bonusNumber);
```
간결하게 표현 가능하겠네요, 다른 policy 구현 객체도 마찬가지구요